glide.yaml

```diff
@@ -181,6 +181,13 @@ import:
 - package: google.golang.org/appengine
   version: 12d5545dc1cfa6047a286d5e853841b6471f4c19
 
+  # to avoid go4.org website outages
+- package: go4.org
```

Comment:
this mitigates the issues from having go4.org site down... see: https://github.com/kubernetes/kubernetes/issues/39868